### PR TITLE
fix: correctly support command complete in postgres-extended

### PR DESCRIPTION
**Pull request by skyzh**
Signed-off-by: Alex Chi <iskyzh@gmail.com>

This PR also updates the test case in correspondence with pg15.

close https://github.com/risinglightdb/sqllogictest-rs/issues/142

### CODE REVIEW

**Comment by skyzh** [comment]
We can wait until the upstream PR gets merged before proceeding.

**Comment by skyzh** [comment]
Shall we fork rust-postgres and release our own fork for now?

**Comment by skyzh** [comment]
But some crates rely on Postgres-types... I'm not sure if we can override dependency when releasing this crate on crates.io.

**Comment by xxchan** [comment]
👀 What's wrong? Isn't the PR merged?

**Comment by skyzh** [comment]
Need wait for upstream to release new bersion

**Comment by wangrunji0408** [comment]
> I'm not sure if we can override dependency when releasing this crate on crates.io.

I'm afraid not. Unless we also publish a patched version of the dependency. 🤣

**Comment by skyzh** [comment]
cc @xxchan @wangrunji0408 ready for review

File: sqllogictest-engines/src/postgres/postgres_extended_test.slt
Comment (xxchan) [inline sqllogictest-engines/src/postgres/postgres_extended_test.slt:1]: How is this change related with the code change?

Comment (xxchan) [reply sqllogictest-engines/src/postgres/postgres_extended_test.slt:1]: & Could you please update the PR description to include all the changes?

Comment (skyzh) [reply sqllogictest-engines/src/postgres/postgres_extended_test.slt:1]: This is probably not related with the code change. I’m using pg15 and probably yields different result than before?

Comment (skyzh) [reply sqllogictest-engines/src/postgres/postgres_extended_test.slt:1]: Updated the PR body 🤪

**Comment by skyzh** [comment]
ping for @xxchan for a final review if you have time :) Thanks!

**Comment by xxchan** [comment]
Sorry I'm too lazy. I will take a look tonight.

**Review by xxchan — APPROVED** [review]
good

**Pull request merged**
